src/SUMMARY.md

```diff
@@ -29,6 +29,7 @@
     - [Emitting Errors and other Diagnostics](diagnostics.md)
         - [`LintStore`](./diagnostics/lintstore.md)
         - [Diagnostic Codes](./diagnostics/diagnostic-codes.md)
+    - [Command-line arguments](./cli.md)
```

Comment:
I would suggest moving this to the top of Part 3, next to the rustc-driver chapter.